plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -457,28 +460,24 @@ private FileWriter createParquetFileWriter(String path)
         try {
             Closeable rollbackAction = () -> fileSystem.deleteFile(path);
 
-            List<Type> parquetTypes = dataColumnTypes.stream()
```

Comment:
This change belongs rather to the previous commit

Reply:
fixed